server/api/redirector.go

```diff
@@ -93,21 +95,21 @@ func (p *customReverseProxies) ServeHTTP(w http.ResponseWriter, r *http.Request)
 
 		resp, err := p.client.Do(r)
 		if err != nil {
-			log.Error(err)
+			log.Error(fmt.Sprintf("%+v", err))
```

Comment:
This really confused me. When do we use a zap.Field, and when do we use fmt.Sprint?